### ISSUE TITLE
Refactor Max/Min to High/Low  

### ISSUE THREAD

**Issue by jansoren** [ta4j/ta4j#300]
Refactor Max/Min to High/Low  to keep the OHLC standard

https://github.com/ta4j/ta4j/blob/master/ta4j-core/src/main/java/org/ta4j/core/Bar.java#L46

**Comment by team172011** [comment]
I agree with you. In your PR #294 please take a look at my comment for `Bar#Num getPrice(PriceType priceType);`

**Comment by milczarekIT** [comment]
Please close this issue. It is done: https://github.com/ta4j/ta4j/pull/326

**Comment by milczarekIT** [comment]
PR is merged, but my question is: why only method was refactored? In my opinion fields minPrice, maxPrice also should be renamed
